<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadWidget</class>
 <widget class="QWidget" name="UploadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Select the videos and subtitles  (only subtitles will be uploaded):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QToolButton" name="buttonUploadBrowseFolder">
          <property name="icon">
           <iconset>
            <normaloff>:/images/openfolder.png</normaloff>:/images/openfolder.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonUploadPlusRow">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonUploadMinusRow">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonUploadDeleteAllRow">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/delete_all.png</normaloff>:/images/delete_all.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonUploadUpRow">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="buttonUploadDownRow">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>401</width>
            <height>33</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="UploadListView" name="uploadView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="uploadDetailsWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="labelReleaseName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Release name:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QCheckBox" name="comboAutomaticTranslation">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="labelTranslator">
             <property name="text">
              <string>Translator</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QCheckBox" name="comboHearingImpaired">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="LanguageComboBox" name="uploadLanguages"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="labelHearingImpaired">
             <property name="text">
              <string>Hearing impaired</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Provider:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="buttonUploadFindIMDB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Find</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="4">
            <layout class="QHBoxLayout" name="horizontalLayout_2"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelMovieTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Movie Title:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="3">
            <widget class="QPushButton" name="buttonUpload">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Upload</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/images/upload.png</normaloff>:/images/upload.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="9" column="2" colspan="2">
            <widget class="QLineEdit" name="uploadTranslator"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelSubtitleLanguage">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Subtitle Language:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QCheckBox" name="comboHighDefinition">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QLineEdit" name="uploadReleaseText"/>
           </item>
           <item row="11" column="2" colspan="2">
            <widget class="QTextEdit" name="uploadComments">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="uploadIMDB">
             <item>
              <property name="text">
               <string>Click on the Find button to identify the movie</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="labelAutomaticTranslation">
             <property name="text">
              <string>Automatic translation</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelHighDefinition">
             <property name="text">
              <string>High Definition</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="labelComments">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Comments:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="ProviderComboBox" name="comboProvider"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UploadListView</class>
   <extends>QTableView</extends>
   <header>subdownloader.client.gui.views.upload</header>
  </customwidget>
  <customwidget>
   <class>LanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>subdownloader.client.gui.views.language</header>
  </customwidget>
  <customwidget>
   <class>ProviderComboBox</class>
   <extends>QComboBox</extends>
   <header>subdownloader.client.gui.views.provider</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
